### PR TITLE
Add dedication

### PR DESCRIPTION
**Pull request by szimmer**

### CODE REVIEW

**Comment by szimmer** [comment]
Hold off on review of this for now

**Comment by szimmer** [comment]
@rpowell22  @ivelasq  Now a brief, unified dedication

File: index.Rmd [hunk 1]
```diff
@@ -73,6 +73,23 @@ print_gt_book <- function(gtobj, ref){
...
-# Preface {-}
+`r if (knitr:::is_html_output()) '# Dedication {-}'`
 
-<!-- TODO: What do we want here? Anything? Maybe a bit on what people should install before using and how to get data from OSF? Acknowledgements? -->
+```{r}
+#| label: index-dedication-text
+#| echo: false
+#| results: asis
+
+thanks <- "To Will, Tom, and Drew, thanks for all the help with additional chores and plenty of git consulting."
```
Comment (ivelasq) [inline index.Rmd:83]:
```suggestion
thanks <- "To Will, Tom, and Drew, thanks for all the help with additional chores and plenty of Git consulting."
```

**Comment by ivelasq** [comment]
Looks great @szimmer , I checked the HTML and PDF versions. My only edit is to capitalize Git.

**Pull request merged**